ethers-ext/src/core/util.ts

```diff
@@ -43,3 +43,175 @@ export const HexStr = {
     return ethers.utils.hexZeroPad( value, length);
   }
 };
+
+// Klaytn Type Enumeration
+export enum Klaytn {
```

Comment:
What about having two enums?
- enum TxType { ValueTransfer = 8, ... }
- enum AccountKeyType { Legacy = 1, ... } 

"Klaytn" is way too generic.

Reply:
How about not using enum (3rd suggestion without '.' ) ? 

e.g. 
1. now :  Klaytn.TxTypeValueTransfer 
2. your suggestion :  TxType**.**ValueTransfer 
3. another suggestion :  TxTypeValueTransfer

Reply:
@nohkwak Is it possible to import a single line when we want to import several tx types?

Reply:
Good point. 
So I updated 2nd suggestion(https://github.com/klaytn/web3klaytn/pull/440/commits/3e9d9a1f6d2956d6c67d90ae9ff410232e9ee2ea).